content/creator/sdk7/networking/authoritative-servers.md

```diff
@@ -0,0 +1,71 @@
+
+{{< hint info >}}
+**💡 Tip**: It's also possible to opt for a hybrid approach where changes are notified between players via Messagebus messages, but the final state is also stored via an API for future visitors.
+{{< /hint >}}
+
+- Websockets: This alternative is more robust, as it establishes a two-way communications channel between player and server. Updates can be sent from the server, you could even have game logic run on or validated on the server. This enables real time interaction and makes more fast paced games possible. It's also more secure, as each message between player and server is part of a session that is opened, no need to validate each message.
+
+## Example scenes with authoritative server
+
+- [API + DB](https://github.com/decentraland-scenes/Awesome-Repository#use-an-api-as-db)
+
+- [Websockets](https://github.com/decentraland-scenes/Awesome-Repository#websockets)
+
+## Preview scenes with authoritative servers
+
+To preview a scene that uses an authoritative server, you must run both the scene and the server it relies on. The server can be run locally in the same machine as the preview, as an easier way to test it.
+
+To start the server, go to the `/server` folder and run `npm run start`.
+
+Once the server is running, either remotely or locally, you can run `npm run start` on the scene as you normally do for local scenes.
+
+Once the scene preview is running, you can open multiple browser tabs pointing at the same local address. Each tab will instantiate a separate player in the same scene, these players will share the same scene state as the scene changes.
+
+See [preview a scene]({{< ref "/content/creator/sdk7/getting-started/preview-scene.md" >}}) for more details.
+
+## Separate realms
+
+Players in decentraland exist in many separate _realms_. Players in different realms cant see each other, interact or chat with each other, even if they're standing on the same parcels. Dividing players like this allows Decentraland to handle an unlimited amount of players without running into any limitations. It also pairs players that are in close regions, to ensure that ping times between players that interact are acceptable.
+
+If your scene sends data to a 3rd party server to sync changes between players in real time, then it's important that changes are only synced between players that are on the same realm. You should handle all changes that belong to one realm as separate from those on a different realm. Otherwise, players will see things change in a spooky way, without anyone making the change.
```

Comment:
```suggestion
If your scene sends data to a server to sync changes between players in real time, then it's important that changes are only synced between players that are on the same realm. You should handle all changes that belong to one realm as separate from those on a different realm. Otherwise, players will see things change in a spooky way, without anyone making the change.
```